test/source/symlink-test/symlinked-file-outside-source

```diff
@@ -1 +1 @@
-/etc/passwd
\ No newline at end of file
+C:/etc/passwd
```

Comment:
This doesn't match the PR description. Is it a deliberate change?

Reply:
I reverted.